feature_extractor/entity_helper.py

```diff
@@ -1,19 +1,22 @@
-
 from typing import Callable, Dict, Tuple
-import nmmo
+
 import numpy as np
+
+import nmmo
 from nmmo.entity.entity import EntityState
 
 from feature_extractor.target_tracker import TargetTracker
-from model.util import one_hot_generator
+
 from team_helper import TeamHelper
 
+from model.util import one_hot_generator
+from model.model import ModelArchitecture
+
 EntityAttr = EntityState.State.attr_name_to_col
 
-N_SELF_FEATURE = 262
-PER_ENTITY_FEATURE = 29
-N_NPC_CONSIDERED = 9
-N_ENEMY_CONSIDERED = 9
+PER_ENTITY_FEATURE = ModelArchitecture.n_ent_feat
```

Comment:
Let's just reference these directly, rather than assigning them to local const

Reply:
Removed the local consts in both the `entity_helper.py` and `map_helper.py`. Will do so going forward.